j2me/communication/src/org/javarosa/service/transport/securehttp/AuthenticatedHttpTransportMessage.java

```diff
@@ -241,6 +250,30 @@ public void send() {
             }
         }
     }
+    
+    private void attemptBrowserCertificateAcceptCallout() {
+    	try {
+	    	if(CalloutResolver != null) {
```

Comment:
It's something that needs to get wired manually into the MIDlet (and would be optional for someone using the JR libraries), so shouldn't need to warn.